server/src/test/java/io/crate/integrationtests/LuceneQueryBuilderIntegrationTest.java

```diff
@@ -313,7 +314,10 @@ public void testWhereNotEqualAnyWithLargeArray() throws Exception {
 
     @Test
     public void testNullOperators() throws Exception {
-        DataType<?> type = randomType();
+        DataType<?> type;
+        do {
+           type = randomType();
+        } while (type == DataTypes.INTERVAL);
```

Comment:
I still don't really undestand why this change became necessary within this PR? That interval cannot be stored isn't new. Did `randomType` change in some unrelated way?